scaffold-working-dir/action.yaml

```
@@ -18,7 +18,7 @@ runs:
     - run: mkdir -p "$(dirname "${{steps.target-config.outputs.working_directory}}")"
       shell: bash
 
-    - run: cp -R "${{steps.target-config.outputs.template_dir}}" "${{steps.target-config.outputs.working_directory}}"
+    - run: cp -R "${{steps.target-config.outputs.template_dir}}/." "${{steps.target-config.outputs.working_directory}}"
```

skip case. this is more better?

```suggestion
    - run: |
        if [ -d "${{steps.target-config.outputs.working_directory}}" ]; then
          cp -R "${{steps.target-config.outputs.template_dir}}" "${{steps.target-config.outputs.working_directory}}"
        else
          :
        fi
```

---

In this way, the workflow create pull request with `no-changes` label. It seems like good.

---

The `if` condition seems to be wrong.

AS IS

```sh
if [ -d "${{steps.target-config.outputs.working_directory}}" ];
```

TO BE

```sh
if [ ! -d "${{steps.target-config.outputs.working_directory}}" ];
```

And `else` isn't needed.

---

That's right ! fixed it.